docker/docker-compose.yml

```diff
@@ -33,24 +35,21 @@ services:
       - GIRDER_MONGO_URI=mongodb://mongo:27017/girder
       - GIRDER_ADMIN_USER=${GIRDER_ADMIN_USER}
       - GIRDER_ADMIN_PASS=${GIRDER_ADMIN_PASS}
+      - CELERY_BROKER_URL=amqp://guest:guest@rabbit/
+      - BROKER_CONNECTION_TIMEOUT=2
 
   girder_worker:
+    runtime: 'nvidia'
```

Comment:
Also, what's requried to run this on a machine without an nvidia GPU? Is just commenting this out sufficient?

Reply:
I don't know.  I sort of doubt it will work at all.

Reply:
It does seem to work (at least, I can run tasks no issue, gpu specific pipelines will probably fail). Maybe an override file like we have for `dev` would be good for people (like myself) trying to run this locally on non-gpu-enabled machines.

Reply:
If you want I can add this, since I've already got something locally

Reply:
We have CPU and GPU binaries of VIAME which have slightly different pipelines enabled, some pipelines will fail some wont in machines without GPUs if you install the GPU version on them